plugins/cpu/README.md

```diff
@@ -0,0 +1,9 @@
+CPU Microcde Support
+=================
+
+Introduction
+------------
+
+This plugin reads the sysfs attributes associated with CPU microcode.
+It displays a read-only value of the CPU microcode version loaded onto
+the physical CPU at fwupd startup.
```

Comment:
maybe a stupid question; but does the microcode have to be the same version on all cores and for all CPUs on the system?

Reply:
For the physical CPU package yes.  That's why I'm only making the device when physical ID changes.